docs/UserGuide.md

```diff
@@ -118,6 +120,34 @@ Example:
 * `task A0123456Z`
     * Lists out the tasks that student (A0123456Z) has.
     
+### Marking an undone task as done for a particular student: `mark`
+
+Marks a specific undone task as done for a particular student.
+
+Format : `mark n/STUDENT_NAME UNDONE_TASK_INDEX` (or) `mark i/STUDENT_ID UNDONE_TASK_INDEX`
+
+* The undone task corresponding to the index or the particular student will be marked as done in the TAPA.
+  
+Example:
+* `mark n/John 1`
+    * Marks the first undone task for John as done.
+* `mark i/AXXXXXXXR 1`
+    * Marks the first undone task for the student with student ID AXXXXXXXR as done.
+
+### Marking a done task as undone for a particular student: `unmark`
+
+Marks a specific done task as undone for a particular student.
+
+Format : `unmark n/STUDENT_NAME DONE_TASK_INDEX` (or) `unmark i/STUDENT_ID DONE_TASK_INDEX`
+
+* The done task corresponding to the index for the particular student will be marked as undone in the TAPA.
```

Comment:
Might want to include what the user should expect if an invalid index is supplied (maybe an error message or smt?)